### ISSUE TITLE
Root cause fix for end of line standardization

### ISSUE THREAD

**Issue by akachru-github** [icgc-argo/argo-docs#472]
After fixing specific dictionary 1.13 diffs issue: https://github.com/icgc-argo/argo-docs/issues/468
Research a way to sanitize the dictionary changes to avoid having this problem in the future. 

Proposal:
- Update scripts to do sanitization
- Update Dictionary Viewer to handle this

**Comment by ciaranschutte** [comment]
@ciaranschutte ensure root dict version is fixed in docs too for further diffs. release dictionary can't be updated so will need to add manually exception in code.

**Comment by ciaranschutte** [comment]
this is covered by https://github.com/overture-stack/lectern/commit/556284e6da04afde5e38abb1f7e45dc38b7e4c4b in lectern and https://github.com/icgc-argo/argo-docs/pull/474/files in argo docs

**Comment by ciaranschutte** [comment]
@kcullion any created dictionaries or updates to schemas with a `script` restriction should return return the response with the `script` normalized which is to say the code in it containing \r\n should be \n and \n should still be \n

I have tested the following on QA successfully: 
- new dict
- update dict with new schema
- update existing file schema

 easiest way I found was to upload a very small schema something like: 
```
{
  "schemas": [
    {
      "name": "donor",
      "description": "Donor Entity",
      "fields": [
        {
          "name": "cause_of_death",
          "valueType": "string",
          "description": "Cause of Donor Death",
          "restrictions": {
            "script": [
              "(function(){\r\nreturn true;\r\n\r\n\r\n\r\n}())"
            ]
          }
        },
        {
          "name": "gender",
          "valueType": "string",
          "description": "Donor Biological Sex",
          "restrictions": {
            "codeList": [
              "Male",
              "Female",
              "Other"
            ]
          }
        },
        {
          "name": "ethnicity",
          "valueType": "string",
          "description": "Self described",
          "meta": {
            "default": "Unknown"
          }
        }
      ]
    }
  ],
  "name": "Ciaran Test Dictionary",
  "version": "1.4"
}
```

please let me know if I can help with your testing or if you'd like me to move this to closed.

**Comment by ciaranschutte** [comment]
@akachru-github lectern fix in rc 1.13.0 merged & tested
https://github.com/overture-stack/lectern/pull/152
https://github.com/overture-stack/lectern/pull/151

**Comment by kcullion** [comment]
Confirmed with @ciaranschutte ... this can be closed. it's a lectern issue which was fixed.
